serverpkg/admin.py

```diff
@@ -108,15 +108,19 @@ def purge_failed_jobs():
 @login_required
 def get_job_results():
     import re
+    import datetime
     from .server import _job_status_db
     csv_output = "ID , status\n"
     for j in _job_status_db.jobs().values():
-        matches = re.findall(r"(\d{8,9})_(\d{8,9})", j.filename)
+        # matches = re.findall(r"(\d{8,9})_(\d{8,9})", j.filename)
+        matches = j.filename.split('_')
```

Comment:
the name format is "hw2_3456_678.zip" so you will get {2 , 3456, 678 }.
This is why I checked that the values are 8 or 9 digits long